src/chapters/thymeleaf-views/create-template.rst

```diff
@@ -0,0 +1,189 @@
+   </div>
+
+The sections below outline the main ideas presented in the video. However, the
+text is NOT a substitute for completing the work described in the clip.
+
+Template Location
+^^^^^^^^^^^^^^^^^^
+
+In IntelliJ, store templates in the ``templates`` folder, which you should
+place within the ``resources`` directory. As you will see in the video,
+Thymeleaf streamlines finding required templates by using
+``resources/templates`` as the default location.
+
+.. figure:: ./figures/templatesFolder.png
+   :alt: Locate the ``templates`` folder.
+
+Add a Template
+^^^^^^^^^^^^^^^
+
+To create a new template, right-click on the ``templates`` folder and select
+*New --> HTML file*. Give your template a descriptive name, and note that
+IntelliJ provides some boilerplate code:
+
+.. sourcecode:: html
+   :linenos:
+
+   <!DOCTYPE html>
+   <html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Title</title>
+   </head>
+   <body>
+
+   </body>
+   </html>
+
+To tie in the Thymeleaf information, you need to add a URL inside the ``html``
+tag on line 2:
+
+.. sourcecode:: html
+
+   <html lang="en" xmlns:th="https://www.thymeleaf.org/">
+
+The ``xmlns:th`` attribute pulls in information from ``thymeleaf.org`` about
+the keywords and methods that we will use with Thymeleaf. This allows IntelliJ
+to properly apply syntax highlighting and error reporting.
+
+A side effect of the ``xmlns:th`` attribute is that it implements stricter
+requirements for closing HTML tags. In HTML5, we can get away with leaving out
+the ``/`` character in standalone tags like ``<input>``. However, with the
+xml format in Thymeleaf, we must include the character.
+
+For example, we need to update line 4 in the boilerplate HTML code to close
+the ``meta`` tag:
+
+.. sourcecode:: html
+
+   <meta charset="UTF-8" /> <!-- Note the closing '/' character -->
+
+Thymeleaf Template
+-------------------
+
+As described in the video, you can save yourself some time by creating your own
+boilerplate code for a Thymeleaf template. This will save you from having to
+make the edits described above every time you add a new base html file.
+
+#. Right-click on the ``templates`` folder (or any other directory), and select
+   *New --> Edit File Templates*.
+#. In the window that pops up, click the "+" icon to add a new file.
+
+   .. figure:: ./figures/createNewTemplate.png
+      :alt: Create a new template window.
+      :scale: 80%
+
+#. Name your template, set the extension as ``html``, then edit the starter
+   code. This will be the boilerplate HTML that appears anytime you select your
+   custom template. For Thymeleaf, the code should at least close the ``meta``
+   tag and include the ``xmlns`` attribute.
+
+   .. figure:: ./figures/thymeleafTemplateCode.png
+      :alt: Generic Thymeleaf starter code.
```

Comment:
Alt text needs to describe the picture.

Reply:
Fixed. Thanks.  (I'll try to be more careful with future figures).